src/components/DateRangePickerInput.jsx

```diff
@@ -45,6 +45,7 @@ const propTypes = forbidExtraProps({
   readOnly: PropTypes.bool,
   showCaret: PropTypes.bool,
   showDefaultInputIcon: PropTypes.bool,
+  inputIconPosition: PropTypes.oneOf(['before', 'after']),
```

Comment:
Since we reuse this, we probably want to save this as a shape, similar to `OrientationShape`